src/main/groovy/org/ods/doc/gen/core/api/HealthController.groovy

```diff
@@ -17,42 +17,39 @@ class HealthController {
     private HtmlToPDFService htmlToPDFService
 
     @Inject
-    HealthController(HtmlToPDFService htmlToPDFService){
+    HealthController(HtmlToPDFService htmlToPDFService) {
         this.htmlToPDFService = htmlToPDFService
     }
 
-    @GetMapping( "/health")
-    Map check( ) {
+    @GetMapping("/health")
+    Map check() {
         log.info("health check to verify a pdf can be generated, executed")
         generatePdfData()
         Map result = [
                 service: "docgen",
-                status: "passing",
-                time: new Date().toString()
+                status : "passing",
+                time   : new Date().toString()
         ]
 
         return result
     }
 
-    private byte[] generatePdfData() {
+    private generatePdfData() {
```

Comment:
void ?

Reply:
Yes, is a Health controller and we weren't using the result.